### PR TITLE
Make async text cover service bindings too

### PR DESCRIPTION
**Pull request by Samze**

### CODE REVIEW

File: spec.md [hunk 1]
```diff
@@ -653,11 +653,11 @@ If the query parameter described above is present, and the Service Broker
 executes the request asynchronously, the Service Broker MUST return the
 asynchronous response `202 Accepted`.
 
-An asynchronous response triggers the Platform to poll the endpoint
-`GET /v2/service_instances/:instance_id/last_operation` until the Service Broker
+An asynchronous response triggers the Platform to poll the Service Instance 
+or Service Binding's `last_operation` endpoint until the Service Broker
```
Comment (mattmcneeney) [inline spec.md]: get rid of those spaces

Comment (Samze) [reply spec.md]: Done

**Comment by duglin** [comment]
LGTM

**Comment by fmui** [comment]
LGTM

**Pull request merged**